### DISCUSSION TITLE
Effect on existing jobs with change in job_data_expire_days

### DISCUSSION THREAD

**Discussion by nsmathew** [jhuckaby/Cronicle#682] [General]
## Summary

Does a change in `job_data_expire_days` take effect for the existing jobs with just a cronicle service restart?
https://github.com/jhuckaby/Cronicle/blob/master/docs/Configuration.md#job_data_expire_days

## Steps to reproduce the problem

NA

## Your Setup
Installation directly on a server running Debian Linux. 

### Operating system and version?
Debian GNU/Linux 11 (bullseye)

### Node.js version?

v20.10.0

### Cronicle software version?

Version 0.9.39

### Are you using a multi-server setup, or just a single server?
Single

### Are you using the filesystem as back-end storage, or S3/Couchbase?
Uses S3(Self hosted minio instance) for storage.

### Can you reproduce the crash consistently?
NA

## Log Excerpts
NA

**Comment by jhuckaby** [comment]
Nope, sorry, this only takes effect for new jobs.

(This is fixed in Cronicle v2 (Orchestra), coming out in early 2024.)

**Reply by nsmathew** [reply]
OK noted. Thanks!
